### PR TITLE
Add `oc serviceaccounts create-kubeconfig`

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by smarterclayton** [comment]
@fabianofranz

**Comment by smarterclayton** [comment]
@openshift/cli-review

**Comment by juanvallejo** [comment]
@fabianofranz LGTM

**Comment by juanvallejo** [comment]
@smarterclayton updated docs look fine, but travis seems to be generating outdated docs when checking

**Comment by stevekuznetsov** [comment]
@juanvallejo  can you reproduce it ?

**Comment by juanvallejo** [comment]
@stevekuznetsov 
> can you reproduce it ?

Yeah, see https://github.com/openshift/origin/pull/12058

**Comment by stevekuznetsov** [comment]
@juanvallejo The Travis job in this PR is on some old commit, maybe it's correct for the commit it was checking for? Kick it and see if it happens with the current state of this PR.

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
Updated.

**Comment by fabianofranz** [comment]
Worth adding some `hack/test-cmd.sh` for this.

**Comment by smarterclayton** [comment]
[test] flake "no route to fedora cloud infra"

**Comment by smarterclayton** [comment]
[test] fedora copr

**Comment by smarterclayton** [comment]
Any additional comments?

File: pkg/cmd/cli/sa/create_kubeconfig.go [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+	"github.com/openshift/origin/pkg/cmd/templates"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"github.com/openshift/origin/pkg/serviceaccounts"
+)
+
+const (
+	CreateKubeconfigRecommendedName = "create-kubeconfig"
+
+	createKubeconfigShort = `Create a .kubeconfig file for a service account`
```
Comment (fabianofranz) [inline pkg/cmd/cli/sa/create_kubeconfig.go]: `kubeconfig` instead of `.kubeconfig` in this line.

Comment (fabianofranz) [inline pkg/cmd/cli/sa/create_kubeconfig.go]: Also I'd suggest to s/Create/Generate

File: pkg/cmd/cli/sa/create_kubeconfig.go [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+	createKubeconfigUsage = `%s SA-NAME`
+)
+
+var (
+	createKubeconfigLong = templates.LongDesc(`
+    Generate a kubeconfig file that will utilize this service account.
+
+    The .kubeconfig will reference the service account token and use the current server,
```
Comment (fabianofranz) [inline pkg/cmd/cli/sa/create_kubeconfig.go]: `kubeconfig` instead of `.kubeconfig` in this line

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**